docs/content.zh/docs/deployment/cli.md

```diff
@@ -153,6 +153,35 @@ $ ./bin/flink savepoint \
 Triggering the savepoint disposal through the `savepoint` action does not only remove the data from 
 the storage but makes Flink clean up the savepoint-related metadata as well.
 
+### Creating a Checkpoint
+[Checkpoints]({{< ref "docs/ops/state/checkpoints" >}}) can also be manually created to save the
+current state. To get the difference between checkpoint and savepoint, please refer to
+[Checkpoints vs. Savepoints]({{< ref "docs/ops/state/checkpoints_vs_savepoints" >}}). All that's
+needed to trigger a checkpoint manually is the JobID:
+```bash
+$ ./bin/flink checkpoint \
+      $JOB_ID
+```
+```
+Triggering checkpoint for job 99c59fead08c613763944f533bf90c0f.
+Waiting for response...
+Checkpoint(CONFIGURED) 26 for job 99c59fead08c613763944f533bf90c0f completed.
+You can resume your program from this checkpoint with the run command.
+```
+If you want to trigger a full checkpoint while the job periodically triggering incremental checkpoints,
+please use the `--full` option.
```

Comment:
I would rephrase it to sth like:
> By default this command triggers the same type of checkpoint that is configured for the given job. If you want override this behaviour, you can force either full or incremental checkpoint to be performed via `--full` or `--incremental` flags respectively.
> (code example)

Reply:
After some digging, I found when a job is configured to run with rocksdb and incremental checkpoint disabled, the `--incremental` flag could not take effect. That is because the `RocksNativeFullSnapshotStrategy` does not take `CheckpointOptions` into account when taking async snapshots. So since only `--full` flag works when incremental checkpoint enabled, I only describe this usage here. But I'm ok with your phrasing.

Also, we should reconsider the incremental file-sharing chain in this senario. How does a manually triggered incremental checkpoint reuse the files from previous full checkpoints, which exist in `chk-x` instead of `shared` directory? I think we need a more flexible checkpoint directory layout. WDYT?

Reply:
Ahhhh, ok. Then yes, this description works better.

But in that case, what happens when user selects `--incremental` flag with configured `HashMapStateBackend` or full `RocksDB`? In that case the CLI command should ideally fail (exception probably would have to be thrown from somewhere around `CheckpointCoordinator`?)

Reply:
User selects `--incremental` flag in this case will trigger a full checkpoint. This behavior align with the REST API. I think a possible way is to remove the `--incremental` flag (or just remove it from the help messages and documents). After we really support triggering a incremental checkpoint with configured full checkpointing state backend, we add this flag back.

Reply:
Yep, let's remove `--incremental` flag from the CLI. Adding some runtime check somewhere would make also sense anyway.

Reply:
OK, I'll create another ticket adding some runtime check and remove the INCREMENTAL flag from REST API.

Reply:
Can you post a JIRA ticket here?

Reply:
Sure thing, here: https://issues.apache.org/jira/browse/FLINK-33498 . I will start working on this recently.